packages/config/config/devices/0x019b/z-water2.json

```
@@ -0,0 +1,129 @@
+{
+	"manufacturer": "Heatit",
+	"manufacturerId": "0x019b",
+	"label": "Z-WATER2",
+	"description": "Relay controller",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x020b",
+			"zwaveAllianceId": 4898
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Input 1 Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "2",
+			"label": "Input 2 Calibration",
+			"valueSize": 1,
+			"minValue": -60,
+			"maxValue": 60,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "3",
+			"label": "Temperature Report Interval",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 65535,
+			"defaultValue": 870,
+			"unit": "seconds",
+			"unsigned": true
+		},
+		{
+			"#": "4",
+			"label": "Temperature Report Hysteresis",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"unit": "0.1 °C",
+			"unsigned": true
+		},
+		{
+			"#": "5",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_on_prev",
+			"label": "Power Restore State"
+		},
+		{
+			"#": "6",
+			"label": "Output functions",
```

```suggestion
			"label": "Output Behavior",
```
seems better I think